### ISSUE TITLE
install: cannot stat '/src/build/firmware/firmware/trezor-emulator.bin'

### ISSUE THREAD

**Issue by karel-3d** [trezor/trezor-mcu#424]
Trezor emulator build (build.sh EMU) throws this 

`install: cannot stat '/src/build/firmware/firmware/trezor-emulator.bin'`

Not sure if related to https://github.com/trezor/trezor-mcu/issues/423 ; I am building on macOS

**Comment by karel-3d** [comment]
(however it seems that the emulator, even if I manage to build it, is a linux binary, which is useless on macOS, unlike the Trezor T emulator)

**Comment by karel-3d** [comment]
Oh it seems the error is not important, since the .elf file is still there.

**Comment by matejcik** [comment]
yeah, the error is spurious, i didn't want to spend time on modifying the generic part of the script.

might look into it later

**Comment by Sjors** [comment]
Same error for me on macOS and indeed the .elf file does show up.

**Comment by rafaeltokyo** [comment]
in https://github.com/trezor/trezor-mcu/blob/master/script/fullbuild#L103 change $EMULATOR_FILENAME to $FIRMWARE_FILENAME, same with EMU  the compile generate trezon.bin file

**Comment by rafaeltokyo** [comment]
@karel-3d worked ?
